<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CallWindow</class>
 <widget class="QDialog" name="CallWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>274</width>
    <height>97</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Direct Call</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="1">
    <widget class="QPushButton" name="CancelBtn">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="IpInput">
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Dist IP</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="PortInput">
     <property name="placeholderText">
      <string>Dist Port</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="CallBtn">
     <property name="text">
      <string>Call</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="PortLabel">
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="Label">
     <property name="text">
      <string>Your port :</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>IpInput</tabstop>
  <tabstop>PortInput</tabstop>
  <tabstop>CallBtn</tabstop>
  <tabstop>CancelBtn</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>CancelBtn</sender>
   <signal>clicked()</signal>
   <receiver>CallWindow</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>228</x>
     <y>79</y>
    </hint>
    <hint type="destinationlabel">
     <x>272</x>
     <y>31</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
